src/index.tsx

```diff
@@ -1,15 +1,19 @@
-import React from "react"
-import { createRoot } from "react-dom/client"
-import App from "./App"
-import * as serviceWorkerRegistration from "./serviceWorkerRegistration"
-import reportWebVitals from "./reportWebVitals"
-import "./i18n"
+import { QueryClient, QueryClientProvider } from "@tanstack/react-query";
+import React from "react";
+import { createRoot } from "react-dom/client";
+import App from "./App";
+import "./i18n";
+import reportWebVitals from "./reportWebVitals";
+import * as serviceWorkerRegistration from "./serviceWorkerRegistration";
 
-const container = document.getElementById("root")
-const root = createRoot(container as HTMLElement)
+const container = document.getElementById("root");
+const queryClient = new QueryClient();
+const root = createRoot(container as HTMLElement);
 root.render(
   <React.StrictMode>
-    <App />
+    <QueryClientProvider client={queryClient}>
```

Comment:
nit: better to keep this file unmodified and do the `QueryClientProvider` wrapper in `App.tsx`

Reply:
Sure